tests/core/test_stellargraph.py

```diff
@@ -608,12 +608,19 @@ def f():
 
 
 def example_weighted_hin(is_directed=True):
```

Comment:
The functionality is tested in `test_undirected_hin_neighbor_methods`, now that this graph has been switched over to create the new `StellarGraph` implementation:

https://github.com/stellargraph/stellargraph/blob/91d0aa5a77aaf3f92b9274ba66fe47e0f0f0d171/tests/core/test_stellargraph.py#L664-L667